pydocumentdb/base.py

```diff
@@ -97,9 +103,13 @@ def GetHeaders(document_client,
         headers[http_constants.HttpHeaders.OfferThroughput] = options['offerThroughput']
 
     if 'partitionKey' in options:
-        # if partitionKey value is Undefined, serailize it as {} to be consistent with other SDKs
+        # if partitionKey value is Undefined, serialize it as {} to be consistent with other SDKs
         if options.get('partitionKey') is documents.Undefined:
-            headers[http_constants.HttpHeaders.PartitionKey] = [{}]
+            import sys
```

Comment:
Thanks for pointing out. We will clean this up as part of the Python3 conversion we are currently working on.